iModelCore/BeSQLite/PublicAPI/BeSQLite/BeSQLite.h

```diff
@@ -393,6 +393,21 @@ enum class WalCheckpointMode {
     Truncate=3, /* Like RESTART but also truncate WAL */
 };
 
+enum class AttachFileTypes {

```

Comment:
Looking at other `enum class` types in BeSqlite.h, almost all are singular. Furthermore, all the other "...Type" ones are singular. I feel this should be `AttachFileType`.